packages/build/tests/plugins/tests.js

```diff
@@ -368,7 +368,7 @@ test('Plugin errors that occur during the loading phase are piped to system logs
   t.snapshot(normalizeOutput(output))
 })
 
-test('Plugins have a pre-populated Blobs context', async (t) => {
+test.serial('Plugins have a pre-populated Blobs context', async (t) => {
```

Comment:
Due to parallelism, sometimes when calling `getPort()` by the time the port is used by the test it has been claimed by something else. Making the test run serially is a workaround for this.